src/OrchardCore/OrchardCore.Abstractions/Extensions/JsonSerializerOptionsConfiguration.cs

```diff
@@ -28,5 +28,6 @@ public void Configure(JsonSerializerOptions options)
         options.TypeInfoResolverChain.Add(new PolymorphicJsonTypeInfoResolver(_derivedTypesOptions));
         options.Converters.Add(DynamicJsonConverter.Instance);
         options.Converters.Add(PathStringJsonConverter.Instance);
+        options.Converters.Add(LoginInfoJsonConverter.Instance);
```

Comment:
@hyzx86 this is the line that should impact you. The content serializer we use for yessql uses the JsonSerializerOptions to serialize these options.